FinanceAgent/docker_compose/intel/cpu/xeon/compose_remote.yaml

```
@@ -0,0 +1,18 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
```

try to use common environment and remove below section?
```suggestion
x-common-environment:
  &common-env
      llm_endpoint_url: ${REMOTE_ENDPOINT}
      api_key: ${OPENAI_API_KEY}
```

---

made change